internal/api/v1beta1/app_types.go

```diff
@@ -483,6 +487,19 @@ func (s AppStatus) Condition(t ConditionType) *Condition {
 	return nil
 }
 
+func getUpdatedUnits(weight uint8, target uint16) (int, int) {
```

Comment:
Nice!
it would be very helpful to have a godoc with an example containing inputs and expected outputs:

Reply:
Added.